File: C/1009 - Salário com bônus.c
#include <stdio.h>

int main(){

	char nome[50];
	double fixo, vendas, salario;

	scanf(" %[^\n]s",nome);
	scanf("%lf",&fixo);
	scanf("%lf",&vendas);

	salario = fixo + (vendas * 15 / 100);

	printf("TOTAL = R$ %.2lf\n", salario);

	return 0;
}

File: Go/1009 - Salário com bônus.go
package main

import (
	"fmt"
)

func main(){

	var salario, vendas, total float64
	var nome string

	fmt.Scan(&nome)
	fmt.Scan(&salario)
	fmt.Scan(&vendas)

	total = salario + (vendas*15/100)

	fmt.Printf("TOTAL = R$ %.2f\n", total)

}